Makefile

```diff
@@ -71,6 +71,7 @@ LOCAL_IP_ENV?=$(shell ip route get 8.8.8.8 | head -1 | awk '{print $$7}')
 
 # fail if unable to download
 CURL=curl -sSf
+CURL_S=curl -C - -sSf
```

Comment:
I think with this change, `$CURL` is no longer used anywhere - why don't we drop the `CURL_C` variable and just add the `-C` argument to `CURL`?

Reply:
You‘re right.